code/index.html

```diff
@@ -1,22 +1,87 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@700&family=Montserrat:wght@100;400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Newspaper</title>
+</head>
+<body>
     <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+        <img src="images/elephant-15850.png" width="100px" alt="Logo" class="logga">
+        <div class="meny"><button><img src="images/bars-solid.svg" width="30px" alt="Menu"></button>
+            <ul>
+                <a href="#"><li>POPULAR</li></a>
+                <a href="#"><li>LATEST NEWS</li></a>
+                <a href="#"><li>ANIMALS</li></a>
+                <a href="#"><li>DONATION</li></a>
+            </ul></div>
+        
+        <nav>
+            <a href=""><p>POPULAR</p></a>
+            <a href=""><p>LATEST NEWS</p></a>
+            <a href=""><p>ANIMALS</p></a>
+            <a href=""><p>DONATION</p></a>
+            </nav>
```

Comment:
Very nice how alternate between the 2 different menus in the screen sizes! Great idea 👍